webapp/src/containers/Rhs/projectList/index.tsx

```diff
@@ -99,14 +103,14 @@ const ProjectList = () => {
                                     onClick={handleOpenLinkProjectModal}
                                     className='plugin-btn no-data__btn btn btn-primary project-list-btn'
                                 >
-                                    {'Link new project'}
+                                    {'Link New Project'}
                                 </button>
                             </div>
                         </> :
                         <EmptyState
                             title='No project linked'
                             subTitle={{text: 'You can link a project by clicking the below button.'}}
```

Comment:
I think we should use contants for these.

Reply:
yes, we can refactor texts later as for now we have just used them like this everywhere